docs/help.md

```diff
@@ -0,0 +1,20 @@
+---
+hide:
+  - navigation
+---
+
+# Help
+
+Here's where to go if you need help with Textual.
+
+## Bugs and feature requests
+
+Report bugs via GitHub on the Textual [issues](https://github.com/Textualize/textual/issues) page. You can also post feature requests via Github issues, but see the [roadmap](./roadmap.md) first.
```

Comment:
This says both "GitHub" and "Github" -- the latter should likely be the same as the former.